cmd/bedMerge/bedMerge.go

```diff
@@ -50,9 +49,29 @@ func bedMergeLowMem(infile string, outfile string, mergeAdjacent bool) {
 	exception.PanicOnErr(err)
 }
 
-func bedMergeHighMem(infile string, outfile string, mergeAdjacent bool) {
-	var records []bed.Bed = bed.Read(infile)
-	outList := bed.MergeHighMem(records, mergeAdjacent)
```

Comment:
The code looks good to me but I just wonder if the bed.MergeHighMem function is now removed from the bed package?